wsd/AdminModel.hpp

```diff
@@ -384,6 +388,9 @@ class AdminModel
     void setDefDocProcSettings(const DocProcSettings& docProcSettings) { _defDocProcSettings = docProcSettings; }
 
     static int getPidsFromProcName(const std::regex& procNameRegEx, std::vector<int> *pids);
+    static int getAssignedKitPids(std::vector<int> *pids);
+    static int getUnassignedKitPids(std::vector<int> *pids);
+    static int getKitPidsFromSystem(std::vector<int> *pids);
```

Comment:
Maybe for a follow up patch, but this pointer here is not necessary at all. Especially that getPidsFromProcName makes no sense to call with a nullptr (it will just waste some CPU cycles).

Best to take reference: `std::vector<int>& pids`. No pointers, no null, no issues.

Reply:
It makes sense to call getPidsFromProcName with nullptr. The function returns the number of found processes. It is used here: https://github.com/CollaboraOnline/online/blob/de62c0f359ecb589d8480889092c0b6592fe56ed/wsd/AdminModel.cpp#L1070
and here: https://github.com/CollaboraOnline/online/blob/de62c0f359ecb589d8480889092c0b6592fe56ed/wsd/AdminModel.cpp#L1076

As a consequence, all dependent function will have the same type for this parameter.